tests/test_on_data/test_gin_ecephys.py

```diff
@@ -352,6 +353,57 @@ class TestConverter(NWBConverter):
                 if recording.has_scaled_traces() and nwb_recording.has_scaled_traces():
                     check_recordings_equal(RX1=recording, RX2=nwb_recording, return_scaled=True)
 
+    parameterized_aux_list = [
+        param(
+            data_interface=SpikeGLXNIDQInterface,
+            interface_kwargs=dict(file_path=str(DATA_PATH / "spikeglx" / "Noise4Sam_g0" / "Noise4Sam_g0_t0.nidq.bin")),
+        )
+    ]
+
+    @parameterized.expand(input=parameterized_recording_list, name_func=custom_name_func)
+    def test_aux_recording_extractor_to_nwb(self, data_interface, interface_kwargs, case_name=""):
```

Comment:
I had initially included this in the `test_recording_extractor_to_nwb` list but the behavior is slightly different.

Also had to debug one functionality in the Converter usage that differs from stand-alone interface conversion, left a comment next to that corresponding code

Reply:
Also this file is starting to get really cluttered so in a follow-up I think I might break each of these test functions into a separate file for readability